pomSnapshotToLastGitTag.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash 
+#This script will change the vesion of all pom.xml files from:
+#<version>develop-SNAPSHOT</version>
+# to
+#<version>GitTag</version>
```

Comment:
I prefer to set this to `<version>yyyy-mm-dd-hash</version>` as well, just like in pomSnapshotToGitHash.sh  Basically always version the same, is that possible?